app/views/events/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Past events</h2>
+<ul>
+<% Event.past.each do |event| %> 
```

Comment:
- [ ] There should be no application logic in the views -- `Event.past` and `Event.future` are business logic and should not be here.
You are already sending instance variables from `index` action in `events_controller`.
Move the bussines logic in the controller as instance variable and use them directly in here.